### PR TITLE
Creation and management of FAQ (Pre-approved Responses)

### PR DESCRIPTION
**Pull request by imsickofmaps**
Including:

- [ ] Create, update and delete from management UI - supports all of (DEL4)
- [ ] REST API exposure for GET (DEL4.2)
- [ ] Upload batches (DEL12.1)

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@rowanseymour same again - just for notification - not ready for review

File: casepro/backend/__init__.py [hunk 1]
```diff
@@ -199,6 +199,15 @@ def fetch_contact_messages(self, org, contact, created_after, created_before):
                  Message.as_json() for incoming messages and Outgoing.as_json() for outgoing messages.
         """
 
+    @abstractmethod
+    def get_url_patterns(self):
```
Comment (rowanseymour) [inline casepro/backend/__init__.py]: Wondering where we're going with this? Is the actual pre-approved responses stuff in another app?

Comment (imsickofmaps) [inline casepro/backend/__init__.py]: @rowanseymour you can ignore this for now. Sorry, the first stuff for Messaging integration got landed into our develop before we started following the "PR from Epic" approach we agreed.

Comment (rowanseymour) [inline casepro/backend/__init__.py]: I don't think we ever agreed on a "PR from Epic" approach. @ewheeler started [a discussion](https://groups.google.com/forum/#!topic/casepro-dev/fMDUsUbco-I) about our development workflow a while back and we seemed to be agreeing on the opposite - small discrete feature PRs.

Seems like that approach is either for both of us, no?

Also am still curious what these backend URLs are going to be used for.

**Comment by imsickofmaps** [comment]
Closing - will re-PR with combined front and backend FAQ rebased branch coming

**Pull request closed without merging**